AdminWebsite/AdminWebsite/ClientApp/src/app/services/justice-users.service.ts

```diff
@@ -1,7 +1,14 @@
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 import { map, shareReplay } from 'rxjs/operators';
```

Comment:
I know this wasn't added by your PR but `map` is showing as an unused import - could we remove it?

Reply:
Thanks, removed